Chapter1/index.rst

```diff
@@ -1,8 +1,8 @@
 Chapter 1: Understanding Climate Science and Carbon Models
-==========================================================
+===========================================================
 Carbon models are tools for evaluating how disturbances, whether man-made or natural, affect the forest ecosystems surrounding us. Understanding these disturbances provides us with important information about the future of our landscapes and the steps to take to lessen their negative consequences.
 
-**Stocks and flows**
+**Pools, Stocks and flows**
 
 Pools are reservoirs or containers that can hold, accumulate, or discharge chemicals. Nature contains pools in the ocean, the lithosphere, soils, atmosphere, and other places. A carbon pool, for instance, stores and releases carbon. Stock refers to the entire amount of a substance in a pool at any time. 
```

Comment:
**Thought**: Over here, I thought carbon pool was just an example. So my brain tried to skip it. BUT I think carbon pools is what we focus on with FLINT. And it's important to highlight that it's important.

**Question as a reader**: In "Stock refers to the entire amount of a substance in a pool at any time." my thought was what's a 'Substance?' What kind of substance?'. I think we're referring to the chemicals? It would be better to be consistent with the words.

Reply:
um i dont really mean chemicals, i mean like things, for example in a carbon pool, the amount of carbon, in a water pool, the amount of water

Reply:
What about "organic or inorganic matter" instead of chemicals?